packages/x-components/src/x-modules/ai/components/ai-overview.vue

```diff
@@ -315,6 +322,10 @@ export default defineComponent({
       shouldAnimateSuggestion.value = true
     })
 
+    onMounted(() => {
+      $x.emit('AiOverviewMounted', undefined, origin.value)
```

Comment:
The only thing you need to pass as metadata to the event is the `feature`. 
```
 onMounted(() => {
      $x.emit('AiOverviewMounted', undefined, { feature: 'overview' })
    })
```

The `location` is injected by the `bus` itself before emitting to the bus.

https://github.com/empathyco/x/blob/main/packages/x-components/src/composables/use-x-bus.ts#L69